server/utils/presenterUtils.ts

```diff
@@ -36,6 +36,9 @@ interface TwelveHourTimeInputPresenter {
   partOfDay: PartOfDayInputPresenter
 }
 
+export type SortableTableHeaders = { text: string; sort: 'ascending' | 'descending' | 'none' }[]
```

Comment:
Genuine question (not a suggestion) - my understanding of types vs interfaces in Typescript is a little rusty - would making these interfaces have any impact on the code, or are they essentially interchangeable here?

Reply:
and I wonder if this should go in the `viewUtils` alongside the `sortableTable` function?

Reply:
there's very minimal differences now in typescript. see https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

regarding the layout, happy to move them 👍